src/Calculator.js

```diff
@@ -0,0 +1,63 @@
+const { PRICE } = require("./constant/message");
+
+class Calculator {
+  rank = {
+    fifth: 0,
+    fourth: 0,
+    third: 0,
+    second: 0,
+    first: 0,
+  };
+
+  constructor(userLotto, winngingLotto, bonusLotto) {
+    this.userLotto = userLotto;
+    this.winngingLotto = winngingLotto;
+    this.bonusLotto = bonusLotto;
+    this.lotateLotto(userLotto, winngingLotto);
+  }
+
+  static calculateRevenue(rank, purchase) {
+    const totalPrice =
+      rank.first * PRICE.FIRST +
+      rank.second * PRICE.SECOND +
+      rank.third * PRICE.THIRD +
+      rank.fourth * PRICE.FOURTH +
+      rank.fifth * PRICE.FIFTH;
```

Comment:
```suggestion
     Object.keys(rank).forEach((grade) => {
       totalPrice += rank[grade] * PRICE[grade.toUpperCase()];
     });
```
배열의 경우 이런식으로 줄일 수 있을 것 같습니다!

Reply:
아하 저도 이 부분이 마음에 안들었는데 이런 깔끔한 방법이 있었네요!! 감사합니다~